db/schema.rb

```diff
@@ -88,6 +88,7 @@
     t.text "description"
     t.string "kind_of_sport"
     t.boolean "private"
+    t.boolean "single", default: false
```

Comment:
I'm not sure what this is about, is it correct to have this included here?

Reply:
This seems to have resolved itself when the `dev` branch got merged into my feature branch (06385f7). 

Probably nobody commited the applied database changes (`schema.db`) – introduced by the newest migration – when I branched my feature branch off the `dev`. Since then, some other PR has probably commited the `schema.db` and been merged into the `dev`, thus removing my unintentional changes to the `schema.db` file from this PR's diff. 

[Since the `schema.db` file should always represent the database schema in its most recent version](https://stackoverflow.com/a/6450396), I think it's okay to include changes to the `schema.db` file even if the corresponding user story does not touch the database structure. Maybe we can discuss this further if a future PR does touch the `schema.db` again.